backend/src/controllers/questionGroupAttempt.js

```diff
@@ -2,21 +2,78 @@ const db = require('../db');
 
 const getQuestionGroupAttempts = async (req, res) => {
   const {
-    query: { questionGroupId },
+    teacher,
+    query: { questionGroupId, pageSize },
   } = req;
 
-  if (!questionGroupId) {
-    return res.status(400).send({
-      message: 'Please provide one of the following: questionGroupId',
+  let questionGroupAttempts;
+
+  if (questionGroupId) {
+    const questionGroupAttemptWhere = questionGroupId
```

Comment:
dit is niet nodig lijkt me, want als hij in de `if` komt, is questionGroupid sowieso bestaand